tests/testthat/test_compare_raster_w_outputs.R

```diff
@@ -0,0 +1,154 @@
+context("Compare raster data to LDS outputs")
+library(dplyr)
+library(tibble)
+library(raster)
+library(rgdal)
+library(testthat)
+
+
+test_that("Compare raster outputs with LDS outputs at basin level", {
+  #Get land data from raster files for each basin intersection (32 regions, 235 basins)
+  #Get output data from LDS and compare with the above
+  #Allow an error tolerance of 2% to account for rounding. Note that this works fine with an error tolerance of 1%
+  
+  #1. Get LDS data and iso data
+  LDS_data<-read.csv("outputs/Land_type_area_ha.csv",skip = 9, header = T,stringsAsFactors = FALSE)
+  ISO<-read.csv('outputs/iso_GCAM_regID.csv',skip = 6, header=T,stringsAsFactors = FALSE)
+  LDS_data %>%left_join(ISO %>% dplyr:::select(iso,GCAM_region_ID),by="iso") ->LDS_data
```

Comment:
Why the need to return the value of the internal variable with `:::` and not just `::` for what is exported in the namespace of the package?